LayoutTests/imported/w3c/web-platform-tests/html/semantics/interactive-elements/the-summary-element/interactive-content.html

```diff
@@ -0,0 +1,118 @@
+  <svg style="width: 160px; height: 100px" viewBox="0 0 100 100">
+    <a href="#" id="svg_a"><text id="svg_text" x="50" y="90" text-anchor="middle">SVG anchor element</text></a>
+  </svg>
+  <svg style="width: 100px; height: 200px" viewBox="0 0 100 100">
+    <foreignObject x="0" y="60" width="100" height="200" text-anchor="middle">
+      <a xmlns="http://www.w3.org/1999/xhtml" href="#" id="svg_foreignObject_a">SVG foreignObject with HTML anchor element</a>
+    </foreignObject>
+  </svg>
+  <audio id="audio" controls src=/media/sound_5.mp3></audio>
+  <button id=button>button element</button>
+  <embed id=embed src="/images/blue.png" height="100" width="100">
+  <iframe id=iframe srcdoc="iframe element"></iframe>
+  <img id=img_usemap usemap src=/media/poster.png></img>
+  <img id=img src=/media/poster.png></img>
+  <input type="text" value="input@type=text" id="input_text">
+  <input type="search" value="input@type=search" id="input_search">
+  <input type="tel" value="input@type=tel" id="input_tel">
+  <input type="url" value="input@type=url" id="input_url">
+  <input type="email" value="input@type=email" id="input_email">
+  <input type="password" value="input@type=password" id="input_password">
+  <input type="button" value="input@type=button" id="input_button">
+  <input type="reset" id="input_reset">
+  <input type="submit" id="input_submit">
+  <input type="date" value="input@type=date" id="input_date">
+  <input type="month" value="input@type=month" id="input_month">
+  <input type="week" value="input@type=week" id="input_week">
+  <input type="time" id="input_time">
+  <input type="datetime-local" id="input_datetime-local">
+  <input type="color" id="input_color">
+  <input type="number" value="1337" id="input_number">
+  <input type="range" id="input_range">
+  <input type="checkbox" id="input_checkbox">
+  <input type="radio" id="input_radio" disabled>
+  <input type="file" id="input_file">
+  <input type="image" id="input_image" src=/media/poster.png>
+  <label id=label style="display: block">label element</label>
+  <textarea value="textarea" id="textarea">textarea element</textarea>
+  <video id="video" controls>
+    <source src="/media/test-1s.mp4" type="video/mp4">
+    <source src="/media/test-1s.webm" type="video/webm">
+  </video>
+  <div id="non-interactive">This is clickable summary text</div>
+  </summary>
+</details>
+
+<script>
+const details = document.querySelector("details");
+
+function clickElement(targetId, isInteractive) {
+  const target = document.getElementById(targetId);
+  details.open = false;
+  if (target.click) {
+    assert_equals(target.id, targetId);
+    target.click();
+    if (isInteractive) {
+      assert_false(details.open);
+    } else {
+      assert_true(details.open);
+    }
+  }
+  else {
+    // svg element don't have click method
+    target.dispatchEvent(new MouseEvent('click', {
+      view: window,
+      bubbles: true,
+      cancelable: true
+    }));
+  }
```

Comment:
We never assert the details state for the SVG elements.

More-over, I think this should use `await test_driver.click(target);` to emulate an actual native click on the shadow DOM of the input.

Reply:
OK — working on making those test update right now